connectivity/mbedtls/include/mbedtls/psa_util.h

```diff
@@ -86,9 +86,9 @@ static inline psa_algorithm_t mbedtls_psa_translate_cipher_mode(
         case MBEDTLS_MODE_ECB:
             return( PSA_ALG_ECB_NO_PADDING );
         case MBEDTLS_MODE_GCM:
-            return( PSA_ALG_AEAD_WITH_TAG_LENGTH( PSA_ALG_GCM, taglen ) );
+            return( PSA_ALG_AEAD_WITH_SHORTENED_TAG( PSA_ALG_GCM, taglen ) );
```

Comment:
I'd suggest keeping this unchanged, and add the old `PSA_ALG_AEAD_WITH_TAG_LENGTH` back to platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM//TARGET_TFM_LATEST/include/psa/crypto_values.h

We still have a TF-M v1.0 target CYTFM_064B0S2_4343W which has old PSA in platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM//TARGET_TFM_V1_0/

Reply:
We can try to build the TLS examples with CYTFM_064B0S2_4343W too, though everything will eventually tested by the CI

Reply:
Slightly OT: It would be ideal if we ran the "build examples" CI on every change automatically (i.e in a github action) then we could find these issues immediately instead of relying on manual testing.

Reply:
There are many examples and targets (especially targets), and some issues are limited to only one target so the testing is quite extensive.